.github/workflows/cd-release.yml

```
@@ -59,8 +59,8 @@ jobs:
           tag: ${{ steps.bump.outputs.new_tag }}
           body: ${{ steps.generate_release_notes.outputs.body }}
     outputs:
-      VersionNumber: ${{ steps.bump.outputs.new_tag }}
-  
+      VersionNumber: ${{ steps.version.outputs.versionId }}
```

actionlint (1.7.7)

property "version" is not defined in object type {bump: {conclusion: string; outcome: string; outputs: {string => string}}; generate_release_notes: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)